GiwazipClient/Views/PostingTextViewController.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  PostingTextViewController.swift
+//  GiwazipClient
+//
+//  Created by creohwan on 2023/01/31.
+//
+
+import UIKit
+
+import SnapKit
+
+class PostingTextViewController: BaseViewController {
+    
+    // MARK: - Property
+    
+    private let textViewPlaceHolder: String = """
+                                            예시)
+                                            - 화장실이 빨강색이면 좋겠어요~!
+                                            - 집이 커졌으면 좋겠어요~!
+                                            """
+    
+    // MARK: - View
+    
+    private let textGuideLabel: UILabel = {
+        $0.text = "문의 내역을 작성해주세요"
+        $0.numberOfLines = 0
+        $0.textColor = .black
+        $0.font = UIFont.systemFont(ofSize: 18, weight: .semibold)
+        return $0
+    }(UILabel())
+    
+    private lazy var textView : UITextView = {
+        $0.text = textViewPlaceHolder
+        $0.textColor = .white
+        $0.backgroundColor = .lightGray
+        $0.font = UIFont.systemFont(ofSize: 16, weight: .regular)
+        $0.textContainerInset = UIEdgeInsets(top: 14, left: 14, bottom: 14, right: 14)
+        $0.textAlignment = .natural
+        $0.layer.cornerRadius = 8
+        $0.delegate = self
+        return $0
+    }(UITextView())
+    
+    private lazy var finishButton: UIButton = {
+        let buttonConfiguration = UIButton.Configuration.filled()
+        $0.configuration = buttonConfiguration
```

Comment:
p5;
`buttonConfiguration`이 이 안에서 한번만 사용될 것이라면 아래와 같이 한줄로 합치는 것은 어떨까요?
```suggestion
        $0.configuration = UIButton.Configuration.filled()
```

Reply:
> p5; `buttonConfiguration`이 이 안에서 한번만 사용될 것이라면 아래와 같이 한줄로 합치는 것은 어떨까요?

a27ab55 감사합니다~

Reply:
> > p5; `buttonConfiguration`이 이 안에서 한번만 사용될 것이라면 아래와 같이 한줄로 합치는 것은 어떨까요?
> 
> [a27ab55](https://github.com/Samsamhada/Giwazip-Client-App/commit/a27ab55c577dc5f065e61ed1919446fbd76d1cbb) 감사합니다~

확인했습니다. 리뷰 리졸브합니다~